idunn/blocks/wikipedia.py

```diff
@@ -330,6 +336,8 @@ def get_wiki_info(cls, wikidata_id, lang, wiki_index):
 
         wiki = resp[0]['_source']
 
+        es_poi.wiki_resp = wiki
```

Comment:
I am not fond of this solution, especially because it requires this block to be instantiated before other blocks that depends on wikidata details.   
Would it be possible to call `WikidataConnector` directly from `PlaceData` ?